Integrations/src/main/java/io/deephaven/integrations/python/JavaLambdaFactory.java

```diff
@@ -0,0 +1,84 @@
+package io.deephaven.integrations.python;
+
+import org.jpy.PyObject;
+
+import javax.annotation.Nullable;
+import java.lang.invoke.CallSite;
+import java.lang.invoke.LambdaConversionException;
+import java.lang.invoke.LambdaMetafactory;
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodHandleProxies;
+import java.lang.invoke.MethodHandles;
+import java.lang.invoke.MethodType;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * Tool to wrap Python Callables into Java functional interfaces. Useful for Java-aware Python code to implement Java
+ * functional interfaces to pass callbacks/handlers/etc to Python-oblivious Java APIs.
+ */
+public class JavaLambdaFactory {
```

Comment:
Unfortunately, we don't have wiring to run any junits from within a docker container so that python/jpy can be installed and correctly running, at least not for :Integrations. That requires your host OS to run Gradle, calling docker, to run Gradle...